Add 1.19.8 and 1.20.4 releasenotes (#14704)
istio/istio.io: content/en/news/releases/1.20.x/announcing-1.20.4/index.md

  components, the TLS version is restricted to v1.2. The cipher suites are limited to a subset
  of `ECDHE-ECDSA-AES128-GCM-SHA256`, `ECDHE-RSA-AES128-GCM-SHA256`,
  `ECDHE-ECDSA-AES256-GCM-SHA384`, `ECDHE-RSA-AES256-GCM-SHA384`, and ECDH
  curves to `P-256`. These restrictions apply on the following data paths:

    1. mTLS communication between Envoy proxies;
    1. regular TLS on the downstream and the upstream of Envoy proxies (e.g. gateway);
    1. Google gRPC side requests from Envoy proxies (e.g. Stackdriver extensions);
    1. Istiod xDS server;
    1. Istiod injection and validation webhook servers.

  The restrictions are not applied on the following data paths:

    1. Istiod to Kubernetes API server;
    1. JWK fetch from Istiod;
    1. Wasm image and URL fetch from Istio Proxy containers;
    1. ztunnel.

  Note that Istio injector will propagate the value of COMPLIANCE_POLICY to the
  injected proxy container, when set.
  ([Issue #49081](https://github.com/istio/istio/issues/49081))

- **Fixed** an issue where the local client contained incorrect entries in the local DNS name
  table. ([Issue #47340](https://github.com/istio/istio/issues/47340))

> - **Fixed** a bug that made `PeerAuthentication` too restrictive in Ambient mode.

Reviewer: Ambient is not capitalized
```suggestion
- **Fixed** a bug that made `PeerAuthentication` too restrictive in ambient mode.
```

- **Fixed** a bug where `VirtualService` containing wildcard hosts that aren't present in the service registry are
  ignored.
  ([Issue #49364](https://github.com/istio/istio/issues/49364))

- **Fixed** an issue where `istioctl precheck` inaccurately reports the IST0141 message related to resource permissions.
  ([Issue #49379](https://github.com/istio/istio/issues/49379))

> - **Fixed** a bug for IPv6 only clusters that prevented ServiceEntry-based listeners from having correct SNI matches

Reviewer: ```suggestion
- **Fixed** a bug for IPv6 only clusters that prevented ServiceEntry-based listeners from having correct SNI matches.
```

> - **Fixed** a bug when there are more than one service with same host name within same namespace, there could
>   occur `STRICT_DNS cluster without endpoints` error.

Reviewer: ```suggestion
- **Fixed** a bug when there is more than one service with the same host name within the same namespace a `STRICT_DNS cluster without endpoints` error could occur.
```

  ([Issue #49489](https://github.com/istio/istio/issues/49489))

- **Fixed** an issue that when using a delegate in a `VirtualService`, the effective `VirtualService` may not be
  consistent with expectations due to a sorting error.
  ([Issue #49539](https://github.com/istio/istio/issues/49539))

> - **Fixed** a bug where specifying a URI regex `.*` match within a vs HTTP route does not short short-circuit the other

Reviewer: Same as above.

> - **Fixed** a bug where specifying a URI regex `.*` match within a vs HTTP route did not short-circuit the subsequent

Author: cc @hzxuzhonghu to take a look if this modified expression is correct.

Reviewer 2: Right, but nits: 
vs/VirtualService

Author: Makes sense, thanks!

  HTTP routes behind.

> - **Fixed** an issue that `Endpoint` and `Service` in istiod-remote chart do not respect the revision value.

Reviewer: ```suggestion
- **Fixed** an issue where `Endpoint` and `Service` in the istiod-remote chart do not respect the revision value.
```
